tests/api_tests/nodes/test_views.py

```diff
@@ -535,6 +535,707 @@ def test_create_project_invalid_title(self):
+        self.project_two_url = '/{}nodes/{}/'.format(API_BASE, self.project_two._id)
+        self.private_project_url = "/{}nodes/{}/".format(API_BASE, self.private_project_user_one._id)
+
+        self.public_payload = {'data': [{'id': self.project_one._id, 'type': 'nodes'}, {'id': self.project_two._id, 'type': 'nodes'}]}
+        self.private_payload = {'data': [{'id': self.private_project_user_one._id}]}
+
+    def test_bulk_delete_public_projects_logged_in(self):
+        res = self.app.delete_json_api(self.url, self.public_payload, auth=self.user_one.auth)
+        assert_equal(res.status_code, 204)
+
+        res = self.app.get(self.project_one_url, auth=self.user_one.auth, expect_errors=True)
+        assert_equal(res.status_code, 410)
+        self.project_one.reload()
+        self.project_two.reload()
+
+    def test_bulk_delete_public_projects_logged_out(self):
+        res = self.app.delete_json_api(self.url, self.public_payload, expect_errors=True)
+        assert_equal(res.status_code, 401)
+        assert_equal(res.json['errors'][0]['detail'], 'Authentication credentials were not provided.')
+
+        res = self.app.get(self.project_one_url, auth=self.user_one.auth, expect_errors=True)
+        assert_equal(res.status_code, 200)
+
+    def test_bulk_delete_private_projects_logged_out(self):
+        res = self.app.delete_json_api(self.url, self.private_payload, expect_errors=True)
+        assert_equal(res.status_code, 401)
+        assert_equal(res.json['errors'][0]['detail'], 'Authentication credentials were not provided.')
+
+    def test_bulk_delete_private_projects_logged_in_contributor(self):
+        res = self.app.delete_json_api(self.url, self.private_payload, auth=self.user_one.auth, expect_errors=True)
+        assert_equal(res.status_code, 204)
+
+        res = self.app.get(self.private_project_url, auth=self.user_one.auth, expect_errors=True)
+        assert_equal(res.status_code, 410)
+        self.private_project_user_one.reload()
+
+    def test_bulk_delete_private_projects_logged_in_non_contributor(self):
+        res = self.app.delete_json_api(self.url, self.private_payload, auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        assert_equal(res.json['errors'][0]['detail'], 'You do not have permission to perform this action.')
+
+        res = self.app.get(self.private_project_url, auth=self.user_one.auth)
+        assert_equal(res.status_code, 200)
+
+    def test_bulk_delete_private_projects_logged_in_read_only_contributor(self):
+        self.private_project_user_one.add_contributor(self.user_two, permissions=['read'])
+        self.private_project_user_one.save()
+        res = self.app.delete_json_api(self.url, self.private_payload, auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        assert_equal(res.json['errors'][0]['detail'], 'You do not have permission to perform this action.')
```

Comment:
Related to https://github.com/CenterForOpenScience/osf.io/pull/4353/files#diff-7a6d21a50624153d15e63a5f643401c7R44,

It would be good to know where this 403 gets raised.